prep-for-module-3-frontend.markdown

```diff
@@ -10,11 +10,10 @@
 There are a bunch of JavaScript frameworks out there to use. At Turing, we choose to teach **React** - it's one of the most popular frameworks out there. Actually, a lot of developers will call React a library, which it is...but it can also be described as a framework. It's a little confusing, but the lines are blurred for React. Let's just say that React is a library with some rules and conventions to follow.
 
 Complete **one** of the tutorials below to get introduced to React:
-* [Tyler McGinnis's React Tutorial: A Comprehensive Guide to learning React.js in 2018](https://tylermcginnis.com/reactjs-tutorial-a-comprehensive-guide-to-building-apps-with-react/)
-* [React tutorial from the docs](https://reactjs.org/tutorial/tutorial.html)
-* [Original Scrimba React Course](https://scrimba.com/playlist/p7P5Hd)
+* [Tyler McGinnis's React Tutorial: A Comprehensive Guide to learning React.js in 2018](https://ui.dev/react-hooks)
+* [React tutorial from the docs](https://react.dev/learn/tutorial-tic-tac-toe)
+* [Original Scrimba React Course](https://scrimba.com/learn/learnreact) - In M3 the initial focus will be on learning functional components and the utilization of hooks.
```

Comment:
This new course is 12 hours long.  I think its good that its linked but we'll need to give students guidance on what to skip/focus on.   The old course was 6 hours - would it be better to let them keep using that one and have them skip the Class-based parts and really focus on hooks at the end?

Reply:
Added that to the list